저는 따로 테스트 코드를 작성하지 않았지만 테스트가 실패했다고 하네요!

원인을 찾아보니
```
WeatherwearApplicationTests > contextLoads() FAILED
    java.lang.IllegalStateException at DefaultCacheAwareContextLoaderDelegate.java:180
        Caused by: org.springframework.beans.factory.BeanCreationException at AbstractAutowireCapableBeanFactory.java:1826
            Caused by: jakarta.persistence.PersistenceException at AbstractEntityManagerFactoryBean.java:4[31](https://github.com/sb01-team10/sb01-otboo-team10/actions/runs/15838338919/job/44646364195?pr=14#step:5:32)
                Caused by: org.hibernate.exception.JDBCConnectionException at SQLStateConversionDelegate.java:100
                    Caused by: org.postgresql.util.PSQLException at ConnectionFactoryImpl.java:373
                        Caused by: java.net.ConnectException at Net.java:-2
 ```

현재 깃허브 액션 CI 파이프라인 상의 테스트 환경에서 DB 세팅이 되어 있지 않기 때문에 일어나는 문제 같습니다.
어떡해야할까요?

--- a/src/main/java/com/codeit/weatherwear/user/entity/User.java
+++ b/src/main/java/com/codeit/weatherwear/user/entity/User.java
@@ -4,6 +4,7 @@ import com.codeit.weatherwear.location.entity.Location;
 import com.vladmihalcea.hibernate.type.json.JsonType;
 import jakarta.persistence.Column;
 import jakarta.persistence.Entity;
+import jakarta.persistence.EntityListeners;
 import jakarta.persistence.EnumType;
 import jakarta.persistence.Enumerated;
 import jakarta.persistence.FetchType;
@@ -22,11 +23,15 @@ import lombok.NoArgsConstructor;
 import org.hibernate.annotations.JdbcTypeCode;
 import org.hibernate.annotations.Type;
 import org.hibernate.type.SqlTypes;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
 
 @Entity
 @Table(name = "users")
 @Getter
 @NoArgsConstructor
+@EntityListeners(AuditingEntityListener.class)
 public class User {
 
     @Id
@@ -34,10 +39,12 @@ public class User {
     @Column(updatable = false)
     private UUID id;
 
-    @Column(name = "created_at", nullable = false)
+    @CreatedDate
+    @Column(name = "created_at", updatable = false)
     private Instant createdAt;
 
-    @Column(name = "updated_at")
+    @LastModifiedDate
+    @Column(name = "updated_at", updatable = true)
     private Instant updatedAt;
 
     @Column(nullable = false, unique = true)

